lib/whitehall_importer/integrity_checker/body_check.rb

```diff
@@ -0,0 +1,29 @@
+module WhitehallImporter
+  class IntegrityChecker::BodyCheck
+    attr_reader :proposed_payload, :publishing_api_content
+
+    def initialize(proposed_payload, publishing_api_content)
+      @proposed_payload = proposed_payload
+      @publishing_api_content = publishing_api_content
+    end
+
+    def body_text_matches?
+      proposed_body_text = proposed_payload.dig(:details, :body)
+      publishing_api_body_text = publishing_api_content.dig(:details, :body)
+
+      content_publisher_file_size_selector = ".gem-c-attachment-link .gem-c-attachment-link__attribute:nth-of-type(2)"
+      whitehall_file_size_selector = ".attachment-inline .file-size, .gem-c-attachment-link .gem-c-attachment-link__attribute:nth-of-type(2)"
+      proposed_body_text = remove_html_elements(proposed_body_text, content_publisher_file_size_selector)
+      publishing_api_body_text = remove_html_elements(publishing_api_body_text, whitehall_file_size_selector)
```

Comment:
I feel like this code would be easier to understand if we it was within a method that was called something like `remove_attachment_file_size`

Reply:
Good idea, have moved it to a new private method.